src/renderer/components/outputs/LargeImageOutput.tsx

```diff
@@ -1,14 +1,19 @@
 /* eslint-disable no-nested-ternary */
 import { ViewOffIcon, WarningIcon } from '@chakra-ui/icons';
 import { Box, Center, HStack, Image, Spinner, Text } from '@chakra-ui/react';
-import { memo } from 'react';
+import { Resizable } from 're-resizable';
+import { memo, useEffect, useRef, useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import { useContextSelector } from 'use-context-selector';
+import { useContext, useContextSelector } from 'use-context-selector';
+import { Size } from '../../../common/common-types';
 import { GlobalVolatileContext } from '../../contexts/GlobalNodeState';
+import { SettingsContext } from '../../contexts/SettingsContext';
 import { useDevicePixelRatio } from '../../hooks/useDevicePixelRatio';
+import { useMemoArray } from '../../hooks/useMemo';
+import { DragHandleSVG } from '../CustomIcons';
 import { OutputProps } from './props';
 
-const IMAGE_PREVIEW_SIZE = 200;
+const IMAGE_PREVIEW_SIZE = 216;
```

Comment:
Why this specific number?

Reply:
I removed the 8px of padding from both sides, and added it to the image preview size. I noticed there was a bit you could expand the box without making the node bigger, which was a bit odd. So I just made it bigger by default instead.